README.md

```diff
@@ -19,8 +19,8 @@ Currently the charts are only available through this git repositoy.
 You can install charts using the following commands:
 
 ```bash
-git clone git@github.com:axoflow/syslog-ng-charts.git
-cd syslog-ng-charts
+git clone git@github.com:axoflow/axosyslog-charts.git
+cd axosyslog-charts
 helm install --generate-name charts/syslog-ng-collector
```

Comment:
I think we should call the chart axosyslog-collector.

Reply:
Thanks, fixed!